docs/config.md

```diff
@@ -65,6 +65,7 @@ Within the `mqtt` object, you can add pretty much all the configuration options
 * `reject_unauthorized`
 * `keepalive`
 * `version`
+* `force_disable_retain`
```

Comment:
As mentioned below, I don't think this name is correct.
Also, given that Zigbee2MQTT doesn't have a setting that controls the same functionality, it should probably be mentioned separately in the documentation (with some explanation).